src/app/layouts/BaseLayout.tsx

```diff
@@ -36,7 +36,7 @@ function BaseLayout(props: Props): JSX.Element {
             }
         }
         isLogout && performLogout()
-    }, [logout, isLogout])
+    }, [logout])
```

Comment:
Inside the useEffect(), you are using both `isLogout` and `logout` dependencies. So why declare just `logout` ,  it should be `[logout, isLogout]`

I have made changes to all of these things in the main branch, I don't want to merge this and then overwrite all my changes. Please fix